### ISSUE TITLE
The custom map renderer sample does not work on Android

### ISSUE THREAD

**Issue by silviubogan** [xamarin/xamarin-forms-samples#518]
### Sample

https://github.com/xamarin/xamarin-forms-samples/tree/master/CustomRenderers/Map/Pin

### Description

The same issue is in latest commit in master (df07412), and in the release ZIP file that is linked from the article (https://github.com/xamarin/xamarin-forms-samples/releases/download/78872/Xamarin_Forms___Map_Custom_Renderer.zip)

#### Other relevant links

1. The article with the sample is this: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/custom-renderer/map/customized-pin
2. The download page is this: https://docs.microsoft.com/ro-ro/samples/xamarin/xamarin-forms-samples/customrenderers-map-pin/

### Steps to Reproduce

1. Open the solution
2. Build the .Droid project and run it on a mobile device.

### Expected Behavior

The map loads and is visible.

### Actual Behavior

The Google logo appears, the zoom in/out buttons appear, but the map does not load, not even in 1 minute.

![Screenshot_20190831-192925](https://user-images.githubusercontent.com/198924/64071935-1ae2bc00-cc8e-11e9-8618-421aa0f3f9e2.jpg)

### Information

- IDE and IDE version: Visual Studio 2019 v16.2.3
- Platform with the issue: 
  - Android: 9.0
- Device or simulator/emulator: device.

**Comment by davidbritch** [comment]
Hi @silviubogan 

Have you acquired a Google Maps API key and added it to the Android Manifest?

**Comment by silviubogan** [comment]
@davidbritch No... (I don't remember to have read in the article that I need to do this.)

Thank you.
